### PR TITLE
Add assign seat functionality

### PR DESCRIPTION
**Pull request by Sebastien1999**
This change lets admins manually assign a user to a seat, using the ticket id. Just tap on a seat, fill in a number corresponding to the id of a ticket and click on "assign seat"

### CODE REVIEW

**Review by julian9499 — APPROVED** [review]

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html [hunk 1]
```diff
@@ -409,6 +415,17 @@ <h4>Are you certain you want to delete [[removeAmount]] seat(s) from the followi
...
 
+          assignSeat() {
+            const seat = this.seatToEdit;
+            const assignId = this.assignId;
+            this.$.ajaxAssignSeat.refurl = `/seats/${seat.seatGroup}/${seat.seatNumber}/${assignId}`;
+            this.$.ajaxAssignSeat.generateRequest();
+          }
+
+          onAssignSeat(e, request) {
+            this.$.seatDialog.close();
```
Comment (bramvankooten) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html:449]: Can you add a refresh for the seats here?

**Review by bramvankooten — CHANGES_REQUESTED** [review]
Code seems good, one suggestion though

**Review by bramvankooten — APPROVED** [review]

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html [hunk 2]
```diff
@@ -182,14 +183,19 @@ <h2>Edit seat</h2>
           <div hidden$="[[_checkSeatTaken(seatToEdit)]]">
             <h4>Seat currently taken by:</h4>
             <p>[[seatToEdit.ticket.owner.profile.displayName]]</p>
-            
           </div>
           <div hidden$="[[!_checkSeatTaken(seatToEdit)]]">
             <p>Seat is still available</p>
+            <p>Want to assign a user to a seat?</p>
+            <p>input the ticket ID below</p>
+            <paper-input label="ticket id" value="{{assignId}}"></paper-input>
```
Comment (martijnjanssen) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html]: Can be a one-way binding

```suggestion
            <paper-input label="ticket id" value="[[assignId]]"></paper-input>
```

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html [hunk 3]
```diff
@@ -409,6 +415,18 @@ <h4>Are you certain you want to delete [[removeAmount]] seat(s) from the followi
             this.$.seatDialog.open();
           }
 
+          assignSeat() {
+            const seat = this.seatToEdit;
+            const assignId = this.assignId;
```
Comment (martijnjanssen) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html]: Why not give the variable the actual name? `ticketID`

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html [hunk 4]
```diff
@@ -409,6 +415,18 @@ <h4>Are you certain you want to delete [[removeAmount]] seat(s) from the followi
...
+            this.$.ajaxAssignSeat.refurl = `/seats/${seat.seatGroup}/${seat.seatNumber}/${assignId}`;
+            this.$.ajaxAssignSeat.generateRequest();
+          }
+
+          onAssignSeat(e, request) {
```
Comment (martijnjanssen) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html:443]: We probably want to handle errors here.

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html (hunk 2 above)
Comment (martijnjanssen) [inline src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html]: After the request succeeds, shouldn't we clear this input?

**Review by martijnjanssen — CHANGES_REQUESTED** [review]
Functionality looks fine, but there's some crucial stuff missing to be honest.

File: src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html (hunk 3 above)
Comment (Sebastien1999) [reply src/lancie-admin-endpoint/lancie-admin-pages/lancie-admin-seats/lancie-admin-seats.html]: it has been changed!

**Review by martijnjanssen — APPROVED** [review]

**Pull request merged**